raiden-ts/tests/unit/raiden.spec.ts

```diff
@@ -120,9 +124,109 @@ function makeDummyDependencies(): RaidenEpicDeps {
 }
 
 describe('Raiden', () => {
+  const partner = makeAddress();
+
+  function initEpicMock(action$: Observable<RaidenAction>): Observable<raidenSynced> {
+    return action$.pipe(
+      filter(raidenStarted.is),
+      mapTo(
+        raidenSynced({
+          tookMs: 9,
+          initialBlock: 1,
+          currentBlock: 10,
+        }),
+      ),
+    );
+  }
+
   test('address', () => {
     const deps = makeDummyDependencies();
     const raiden = new Raiden(dummyState, deps, combineRaidenEpics(of(dummyEpic)), dummyReducer);
     expect(raiden.address).toBe(dummyState.address);
   });
+
+  function matrixPresenceEpicMock(action$: Observable<RaidenAction>) {
```

Comment:
Just a nitpick here, it would be good if you keep all the epic mocks at the top together and then start with the tests.

Reply:
Don't you think it gets more readable this way?

Reply:
Ok just that in other tests as well i observed that whatever we wanted to mock we either place it in the `beforeEach` function or just after starting `describe` and then the tests are all grouped together without anything in between each test. but this is minor and you could leave it.

Reply:
I'd prefer, if some mock was specific to a single test and not reused, that this was inside the test body, in order to not pollute the outside context, but this is just a matter of taste and codestyle.

Reply:
✔️